dts/arm/renesas/ra/ra8/ra8x1.dtsi

```diff
@@ -473,6 +473,15 @@
 			status = "disabled";
 		};
 
+		ospi0: ospi@40268000 {
```

Comment:
Consider using:
 
```suggestion
		ospi0: spi@40268000 {
```

to avoid

> node name for SPI buses  should be 'spi'

Reply:
@GeorgeCGV : thanks for your comment and sorry for the late response, 
just a confirmation, may I ask about the app you are building with that CMake warning? I didn't encounter that issue on my side.